windows/src/desktop/kmshell/locale/es-419/strings.xml

```diff
@@ -331,8 +331,8 @@
   <string name="koAutomaticUpdate" comment="Automatically check for updates and download">Buscar actualizaciones y descargar automáticamente</string>
   <!-- Context: Configuration Dialog - Options tab -->
   <!-- String Type: FormatString -->
-  <!-- Introduced: 7.0.230.0 -->
-  <string name="koCheckForUpdates" comment="Startup options - check online weekly for updates">Buscar actualizaciones automáticamente en keyman.com cada semana</string>
+  <!-- Modified: 18.0.96.0 -->
+  <string name="koCheckForUpdates" comment="Automatically check for updates and download">Buscar actualizaciones y descargar automáticamente</string>
```

Comment:
I'm confused - why do we modify `koCheckForUpdates` here but in all other files we removed it?